host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-core-tools.Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM mcr.microsoft.com/dotnet/sdk:5.0
+FROM mcr.microsoft.com/dotnet/sdk:3.1
```

Comment:
Please see https://hub.docker.com/_/microsoft-dotnet-sdk for the right tag

Reply:
Updated in most recent commit to use dotnet/core/sdk:3.1 as it did before PR #515